### PR TITLE
fix #1227 - removed hardcoded resources from xml layout files

### LINKED ISSUE #1227: Remove hardcoded strings from xml layout files
**Summary:**

Remove hardcoded strings resources from xml layout files.

**Steps to reproduce:**

Go through the project codebase and layout files.

**Expected behaviour:**

Remove hardcoded resources from xml layout files.

**Observed behaviour:**

Move hardcoded strings to strings.xml file

**Device and Android version:**

Android Pi.

**Issue for GCI**

Beginner level.

### PR DESCRIPTION
**Pull request by rlichtarski**
fixes #1227

### CODE REVIEW

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -55,17 +71,32 @@
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
 
+    <dimen name="charge_app_widget_min_height">110dp</dimen>
+    <dimen name="charge_app_widget_min_width">180dp</dimen>
+
+    <dimen name="mifos_logo_height">100dp</dimen>
+    <dimen name="mifos_logo_width">100dp</dimen>
+    <dimen name="iv_user_image_height">65dp</dimen>
+    <dimen name="iv_user_image_width">65dp</dimen>
```
Comment (luckyman20) [inline app/src/main/res/values/dimens.xml]: This is repeated, try to give it a more generic name and implement accordingly.

**Review by luckyman20 — CHANGES_REQUESTED** [review]
Check for all the dimensions repeated, update affected files accordingly.

File: app/src/main/res/layout/action_bar_notification_layout.xml [hunk 2]
```diff
@@ -27,10 +27,10 @@
...
         android:layout_height="wrap_content"
         android:layout_width="wrap_content"
-        android:paddingLeft="3dp"
-        android:paddingRight="3dp"
+        android:paddingLeft="@dimen/notification_indicator_padding"
+        android:paddingRight="@dimen/notification_indicator_padding"
         android:textColor="@color/white"
-        android:textSize="10sp"
-        android:visibility="gone"/>
+        android:textSize="@dimen/text_size"
```
Comment (zoman-anamika) [inline app/src/main/res/layout/action_bar_notification_layout.xml]: Can you change text_size to text_size_10sp instead?

Comment (geekanamika) [inline app/src/main/res/layout/action_bar_notification_layout.xml]: Can you change "text_size" to text_size_10sp ?

File: app/src/main/res/layout/beneficiary_spinner_layout.xml [hunk 3]
```diff
@@ -12,13 +12,13 @@
         android:layout_height="wrap_content"
         android:layout_width="match_parent"
         android:textAppearance="?android:textAppearanceMedium"
-        tools:text="1"/>
+        tools:text="@string/one"/>
```
Comment (geekanamika) [inline app/src/main/res/layout/beneficiary_spinner_layout.xml]: It wasn't needed even though it causes no harm :) You are only supposed to shift "android:text" not "tools:text" :)

**Review by geekanamika — APPROVED** [review]

**Comment by geekanamika** [comment]
Good work @rradzzio :) I have added few suggestions for your reference so that you can update PR & learn too :)

**Comment by rlichtarski** [comment]
Thank you so much :) I did what you suggested and added some dimens resources that I missed

**Pull request closed without merging**